tests/test-run.sh

```diff
@@ -199,4 +199,11 @@ printf '%s--dir\0/tmp/hello/world\0' '' > test.args
 $RUN --args 3 test -d /tmp/hello/world 3<test.args
 echo "ok - we can parse arguments from a fd"
 
+PATH="${srcd}:$PATH" $RUN -- sh -c "echo hello" > stdout
+assert_file_has_content stdout hello
+echo "ok - we can run with --"
+PATH="${srcd}:$PATH" $RUN -- --inadvisable-executable-name-- > stdout
```

Comment:
Let's also test that passing say `--chdir /enoent` or something isn't parsed by bwrap.